cmd/publicize/server_test.go

```diff
@@ -433,7 +433,7 @@ and the repository exists.
 			t.Fatalf("test id: %s: %v", tc.id, err)
 		}
 
-		err := s.mergeAndPushToRemote(privateOrg, privateRepo, publicOrg, publicRepo, tc.remoteResolver, tc.branch, false)
+		_, err := s.mergeAndPushToRemote(privateOrg, privateRepo, publicOrg, publicRepo, tc.remoteResolver, tc.branch, false)
```

Comment:
We should not ignore returned values of a method in tests for that method. We cannot predict the SHA but we can test it's a 40-character string when there is no error.